utils/bfd.ml

```diff
@@ -0,0 +1,544 @@
+(**************************************************************************)
```

Comment:
Sure, let us find a better name! To me `cobj` refers more to the `.o` variety of files which is not quite what we are doing here where we look rather to the `.so` or `.exe` variety. Suggestions welcome!

Reply:
Could we consider having one module per file format? Then our modules would be just Elf, Macho, FlexDLL. (For the helper module with the common logic, I vote for "Binutils"!)  This is easier to name and it may have other benefits as well.

Reply:
I like `Binutils`. On the other hand, the whole module is just ~550 lines. Splitting that into 4 modules feels a bit premature.

Reply:
Note: Whether it's 1 or 4 modules we have to worry about compilation-unit-name conflicts with compiler-libs users.

Reply:
>  we have to worry about compilation-unit-name conflicts with compiler-libs users.

`Nojbinutils` ? `Nojbfd` ?  (Just kidding.)

Reply:
> I like `Binutils`. On the other hand, the whole module is just ~550 lines. Splitting that into 4 modules feels a bit premature.

`Binutils` seems fine to me. However if we assume the scope of the module doesn't change we could also go for `Bin_symbols` with an appropriate change of function names so that it reads well (`Bin_symbols.{of_file,mem,offset}`). 

Regarding the splitting I also don't have the feeling it's necessary (and is likely a bit painful given the shared data types), another poor argument against is that it makes it harder to integrate in `Dynlink`'s [mini `compiler-libs`](https://github.com/ocaml/ocaml/blob/trunk/otherlibs/dynlink/Makefile#L72).

Reply:
I renamed to `Binutils`.